Cargo.toml

```diff
@@ -126,8 +128,8 @@ version = "0.77.0"
 edition.workspace = true
 license.workspace = true
 rust-version.workspace = true
-homepage = "https://crates.io"
-repository = "https://github.com/rust-lang/cargo"
+homepage.workspace = true
```

Comment:
Changed from <https://crates.io> to https://github.com/rust-lang/cargo>. Should be acceptable?

Reply:
Good idea. Are we going to link to The Cargo Book for all subcrates or just `cargo` main crate?

Reply:
I think that only makes sense for `cargo`